core/overrides.lua

```diff
@@ -1017,87 +1016,87 @@ end
 --#endregion
 --#region enhancements UI
 function create_UIBox_your_collection_enhancements(exit)
-	local deck_tables = {}
-	local rows, cols = 2, 4
-	local page = 0
-
-	G.your_collection = {}
-	for j = 1, rows do
-		G.your_collection[j] = CardArea(G.ROOM.T.x + 0.2 * G.ROOM.T.w / 2, G.ROOM.T.h, 4.25 * G.CARD_W, 1.03 * G.CARD_H,
-			{
-				card_limit = cols,
-				type = 'title',
-				highlight_limit = 0,
-				collection = true
-			})
-		table.insert(deck_tables,
-			{n = G.UIT.R, config = {align = "cm", padding = 0, no_fill = true}, nodes = {
-				{n = G.UIT.O, config = {object = G.your_collection[j]}}}
-		})
+    local deck_tables = {}
+    local rows, cols = 2, 4
+    local page = 0
+
+    G.your_collection = {}
+    for j = 1, rows do
+        G.your_collection[j] = CardArea(G.ROOM.T.x + 0.2 * G.ROOM.T.w / 2, G.ROOM.T.h, 4.25 * G.CARD_W, 1.03 * G.CARD_H,
+            {
+                card_limit = cols,
+                type = 'title',
+                highlight_limit = 0,
+                collection = true
+            })
+        table.insert(deck_tables, { n = G.UIT.R, config = { align = "cm", padding = 0, no_fill = true },
+            nodes = {{ n = G.UIT.O, config = { object = G.your_collection[j] } }}
+        })
+    end
+
+	table.sort(G.P_CENTER_POOLS.Enhanced, function(a,b) return a.order < b.order end)
```

Comment:
Sort Enhancements when they're initially defined, not here

Reply:
I haven't added this specifically, but I'm pretty sure it's here to make sure the Enhancements are in the correct order before displaying in the collection

Reply:
Ok, I guess we'll just add a TODO comment